### PR TITLE
dbeaver/pro#2706 Usage Statistics dialogue appears after double Workspace reset

### PR DESCRIPTION
**Pull request by serjiokov**
The order of operation is fixed, the current order is:
On startup:
1. Invoke reset (delete) preferences files
2. Invoke bundle preferences initialization

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Review by ShadelessFox — COMMENTED** [review]
Will these log messages be reverted before merging the PR?

Otherwise, please change the logging level to at least `trace` or `debug`

**Comment by MashaKorax** [comment]
App cant start on Linux
[1713994616608.log](https://github.com/dbeaver/dbeaver/files/15101013/1713994616608.log)

**Comment by MashaKorax** [comment]
verified

File: plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/UIStatisticsActivator.java [hunk 1]
```diff
@@ -110,6 +109,9 @@ public static ImageDescriptor getImageDescriptor(String path) {
     }
 
     public DBPPreferenceStore getPreferences() {
+        if (preferences == null) {
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/UIStatisticsActivator.java]: Why? It must be instantiated on bundle activation

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/UIStatisticsActivator.java (hunk 1 above)
Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/UIStatisticsActivator.java]: Generally, avoiding hanging bundle on start and case preferences reset (remove pref files) is better to create preferences at first call.

**Comment by MashaKorax** [comment]
verified

**Pull request merged**